python-package/tests/test_rgf_python.py

```diff
@@ -329,12 +329,37 @@ def test_attributes(self, add_attrs=None):
 
     def test_feature_impotances(self):
         est = self.estimator_class(**self.kwargs)
+        with self.assertRaises(NotFittedError):
+            est.feature_importances_
         est.fit(self.X_train, self.y_train)
 
         fi = est.feature_importances_
         self.assertEqual(fi.shape[0], self.X_train.shape[1])
         self.assertAlmostEqual(fi.sum(), 1)
 
+    def test_warm_start(self):
+        est = self.estimator_class(**self.kwargs)
+        self.assertRaises(NotFittedError, est.save_model, 'model')
+        est.fit(self.X_train, self.y_train)
+        est.save_model('model')
+
+        self.kwargs['init_model'] = 'no_such_file'
+        new_est = self.estimator_class(**self.kwargs)
+        assertRaisesWithRegex = (self.assertRaisesRegex if hasattr(self, 'assertRaisesRegex')
```

Comment:
How about extract here as class method?

Reply:
Sorry, didn't get it. Can you please elaborate?

Reply:
ex.
```
def myAssertRaisesWithRegex(self, *args):
    if sys.version >= '3.0.0':
        self.assertRaisesRegex(args)
    else:
        self.assertRaisesRegexp(args)
```

Reply:
Ah, got it! OK, will do this today.

And thanks a lot for uploading to PyPI!

Reply:
Done!